icubam/backoffice/handlers/icus.py

```diff
@@ -49,12 +50,15 @@ def do_render(self, icu, error=False):
       regions = [e.region for e in self.current_user.managed_icus]
     regions.sort(key=lambda r: r.name)
 
+    icus = self.db.get_managed_icus(self.current_user.user_id)
+    depts = sorted(set([i.dept for i in icus]))
```

Comment:
Can't we get the sorting done by the SQL layer, using an index?

Reply:
Understood. we probably could. However it would probably add quite some complexity on the store side to add a very adhoc feature that does not cost so much to be done in post processing (extract a field and sort)